(node:1447349) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'banner-nosession-42839',
  containerIp: '10.0.3.46',
  mitmPort: 42839,
  autoAccessFixed: 1
}
2024-11-09 04:23:45.862 - [Info] MITM Version: 2
2024-11-09 04:23:45.864 - [Info] Auto Access Enabled: true
2024-11-09 04:23:45.865 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-09 04:23:45.875 - [Info] SSH man-in-the-middle server for 10.0.3.46 listening on 10.0.3.1:42839
2024-11-09 04:24:28.039 - [Debug] [Connection] Attacker connected: 47.237.14.43 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-09 04:24:29.566 - [Debug] [Auth] Attacker 47.237.14.43 trying to authenticate with 'password'
2024-11-09 04:24:29.567 - [Debug] [Auto Access] Attacker: 47.237.14.43, Threshold: 1, Attempts: 1
2024-11-09 04:24:29.567 - [Debug] [Auto Access] Compromising the honeypot
2024-11-09 04:24:29.568 - [Debug] [Auto Access] Adding the following credentials: 'hwangjc:hwangjc'
2024-11-09 04:24:29.894 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-09 04:24:29.896 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-09 04:24:29.896 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.46
2024-11-09 04:24:29.986 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-09 04:24:30.972 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-09 04:24:32.562 - [Debug] [Connection] Attacker connected: 47.237.14.43 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-09 04:24:34.080 - [Debug] [Auth] Attacker 47.237.14.43 trying to authenticate with 'password'
2024-11-09 04:24:34.080 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-09 04:24:34.081 - [Debug] [Auth] Attacker: 47.237.14.43 has so far made 1 attempts. Remaining: 5 attempts
2024-11-09 04:24:34.572 - [Debug] [Connection] Attacker closed the connection
2024-11-09 04:24:34.812 - [Debug] [Connection] Attacker connected: 47.237.14.43 | Client Identification: SSH-2.0-libssh_0.9.6
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
